com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -471,6 +471,7 @@ private static Options getCommandLineOptions() {
         addOption(options, "v", "verbose", false, "Verbose output", false);
         addOption(options, "h", "help", false, "This help message", false);
         addOption(options, "a", "archive", false, "Build archive", false);
+        addOption(options, "na", "no-archive", false, "Ignore resource archives on bundling stage", false);
```

Comment:
is it ok name for such an option? or maybe `ignore-archive` or `ignore-resources`?

Reply:
Ah yes, naming again :-)

I don't like "ignore-resources". Because we do not ignore all resources. We still include the bundle resources for instance.

"no-archive" sounds like the opposite of "archive" but it is not. You could in theory still want to build the archive but not include it in the bundle.

And this is exactly what this option means. We do not include the archive in the bundle. And another way of saying "not include" is to say "exclude", so perhaps "exclude-archive" is better? Possibly also "skip-archive"?

Argh, it is hard!

Reply:
I prefer "exclude-archive"